R/tree.R

```diff
@@ -1,9 +1,12 @@
 #' Print contents of directories in a tree-like format
 #'
 #' @param path A path to print the tree from
+#' @param recursive Should directories be printed recursively?
```

Comment:
```suggestion
#' @inheritParams dir_ls
```

Reply:
It is generally better to inherit params when possible, to avoid duplication of documentation, so if it is changed once it gets changed everywhere.

Reply:
Cool. Can fix

Reply:
@jimhester I've updated this parameter definition now. All the best